src/attributes/classes.js

```diff
@@ -86,7 +84,7 @@ jQuery.fn.extend({
 					}
 
 					// Only assign if different to avoid unneeded rendering.
-					finalValue = value ? jQuery.trim( cur ) : "";
```

Comment:
Probably to save the function call? Probably not worth the extra code. The trimming was added in e76ba32cebc26108e15b2e0e03c5152624c1bad9 to avoid leading/trailing space.